go/pkg/pwdb/helpers.go

```diff
@@ -5,19 +5,24 @@ import (
 
 	"github.com/brianvoe/gofakeit"
 	"github.com/bwmarrin/snowflake"
-	"github.com/jinzhu/gorm"
 	"go.uber.org/zap"
+	"gorm.io/gorm"
 	"pathwar.land/pathwar/v2/go/internal/randstring"
 	"pathwar.land/pathwar/v2/go/pkg/errcode"
 )
 
 func GetInfo(db *gorm.DB, logger *zap.Logger) (*Info, error) {
 	info := Info{
-		TableRows: make(map[string]uint32),
+		TableRows: make(map[string]int64),
```

Comment:
@remyoster Don't we need uint64?

Reply:
As i remember the snowflake IDs are on int64, that's why. 
But indeed it would be better to put all IDs on uint64

Reply:
yes you are right, they are safely generated. we can go with int64